tests/test_core.py

```diff
@@ -186,65 +195,74 @@ def __init__(self, data_home=None, version="default"):
                     "1": core.Index(
                         "blah_1.json", url="https://google.com", checksum="asdf"
                     ),
-                    "2": core.Index("blah_2.json"),
-                    "real": core.Index("beatles_index_1.2.json"),
+                    "real": core.Index("acousticbrainz_genre_index_sample.json"),
+                },
+            )
+
+    class VersionTest2(core.Dataset):
+        def __init__(self, data_home=None, version="default"):
+            super().__init__(
+                data_home,
+                version,
+                indexes={
+                    "default": "2",
+                    "2": core.Index("blah_2.json", url="https://google.com"),
                 },
             )
 
     dataset = VersionTest("asdf")
     assert dataset.version == "1"
-    assert os.path.normpath(dataset.index_path) == os.path.normpath(
-        "asdf/mirdata_indexes/blah_1.json"
-    )
+    assert os.path.join(
+        *dataset.index_path.split(os.path.sep)[-4:]
+    ) == os.path.normpath("mirdata/datasets/indexes/blah_1.json")
 
-    dataset_default = VersionTest("asdf", version="default")
+    dataset_default = VersionTest("asdf")
     assert dataset_default.version == "1"
-    assert os.path.normpath(dataset_default.index_path) == os.path.normpath(
-        "asdf/mirdata_indexes/blah_1.json"
-    )
+    assert os.path.join(
+        *dataset.index_path.split(os.path.sep)[-4:]
+    ) == os.path.normpath("mirdata/datasets/indexes/blah_1.json")
 
     dataset_1 = VersionTest("asdf", version="1")
     assert dataset_1.version == "1"
-    assert os.path.normpath(dataset_1.index_path) == os.path.normpath(
-        "asdf/mirdata_indexes/blah_1.json"
-    )
+    assert os.path.join(
+        *dataset_1.index_path.split(os.path.sep)[-4:]
+    ) == os.path.normpath("mirdata/datasets/indexes/blah_1.json")
     with pytest.raises(FileNotFoundError):
         dataset_1._index
 
     local_index_path = os.path.dirname(os.path.realpath(__file__))[:-5]
     dataset_test = VersionTest("asdf", version="test")
     assert dataset_test.version == "0"
-    assert os.path.normpath(dataset_test.index_path) == os.path.join(
-        local_index_path, os.path.normpath("mirdata/datasets/indexes/blah_0.json")
+    assert dataset_test.index_path == os.path.join(
+        local_index_path, "mirdata/datasets/", "indexes", "blah_0.json"
```

Comment:
change this to `local_index_path, "tests", "indexes", "blah_0.json"` when we decide to move all sample indexes to tests/. CC @genisplaja